dvc/remote/local/__init__.py

```diff
@@ -207,7 +203,7 @@ def walk(self, path_info):
         return os.walk(path_info.path)
 
     def get_file_checksum(self, path_info):
-        return file_md5(path_info.path)[0]
+        return file_checksum(path_info.path, self._checksum_types[0])[0]
```

Comment:
Looks like `file_md5/file_checksum` second result is never used. Maybe no is a good time to drop it and return only hex.